centreon/tests/e2e/features/Dashboards/03-dashboard-properties-edition/index.ts

```diff
@@ -112,8 +114,11 @@ Given(
     cy.visit('/centreon/home/dashboards');
     cy.contains(dashboardToEdit.name)
       .parent()
-      .find('button[aria-label="edit"]')
+      .find('button[aria-label="More actions"]')
       .click();
+
+    cy.get('[aria-label="Edit properties"]').click();
```

Comment:
```suggestion
    cy.getByLabel({ label: 'Edit properties', tag: 'button' }).click();
```
should be applied on every instance of this change